airflow/contrib/operators/dynamodb_to_s3.py

```diff
@@ -32,7 +32,7 @@
 from boto.compat import json  # type: ignore
 
 from airflow.contrib.hooks.aws_dynamodb_hook import AwsDynamoDBHook
-from airflow.models.baseoperator import BaseOperator
+from airflow.models import BaseOperator
```

Comment:
It's mainly because of stability back-forth compatibility but also example for others. If someone develops an external operator or DAG they should use the "airflow.DAG", "airflow.models.BaseOperator". Those are "guaranteed" to be stable. In the future we might decide that for whatever reason we rename the operator to something else and change package, we will still import it here. I really see it as the stable "API" of Airflow.

Regarding the examples - when people create their own operators, they will look at those implemented in core as examples. So better to have them in the way we expect people will be writing them.

Reply:
I hope it explains it :).

Reply:
I think we might be talking the same thing or maybe not :D 

What I was saying is `airflow.operators` and `airflow.contrib.operators` should use same kind of imports e.g "airflow.models.BaseOperator" "airflow.DAG" .

Reply:
I think they are for operators. The change for DAG is coming as follow-up. Have you seen some place where we use it inconsistently ? I think my pre-commit hooks applies the same checks for "operators" and "contrib" and "providers".